clients/rust/marginfi-cli/src/processor/admin.rs

```diff
@@ -0,0 +1,167 @@
+use anchor_client::anchor_lang::prelude::*;
+use anchor_client::anchor_lang::InstructionData;
+use anchor_spl::associated_token;
+use anyhow::Result;
+use marginfi::{
+    bank_authority_seed,
+    state::marginfi_group::{Bank, BankVaultType},
+};
+use solana_sdk::{
+    instruction::Instruction, message::Message, pubkey::Pubkey, transaction::Transaction,
+};
+
+use crate::{
+    config::Config,
+    utils::{process_transaction, ui_to_native},
+};
+
+pub fn process_collect_fees(config: Config, bank_pk: Pubkey) -> Result<()> {
+    let bank = config.mfi_program.account::<Bank>(bank_pk)?;
+    let rpc_client = config.mfi_program.rpc();
+
+    let (liquidity_vault_authority, _) = Pubkey::find_program_address(
+        bank_authority_seed!(BankVaultType::Liquidity, bank_pk),
+        &marginfi::id(),
+    );
+
+    let ix = Instruction {
+        program_id: marginfi::id(),
+        accounts: marginfi::accounts::LendingPoolCollectBankFees {
+            marginfi_group: bank.group,
+            bank: bank_pk,
+            fee_vault: bank.fee_vault,
+            token_program: spl_token::id(),
+            liquidity_vault_authority,
+            liquidity_vault: bank.liquidity_vault,
+            insurance_vault: bank.insurance_vault,
+        }
+        .to_account_metas(Some(true)),
+        data: marginfi::instruction::LendingPoolCollectBankFees {}.data(),
+    };
+
+    let recent_blockhash = rpc_client.get_latest_blockhash().unwrap();
+    let signing_keypairs = config.get_signers(false);
+
+    let message = Message::new(&[ix], Some(&config.authority()));
+    let mut transaction = Transaction::new_unsigned(message);
+    transaction.partial_sign(&signing_keypairs, recent_blockhash);
+
+    match process_transaction(&transaction, &rpc_client, config.get_tx_mode()) {
+        Ok(sig) => println!("Tx succeded (sig: {})", sig),
+        Err(err) => println!("Error:\n{:#?}", err),
```

Comment:
We could `error!()` here and return a non-zero exit code?

Reply:
Maybe we can have a larger PR to rework the CLI a bit, but rn every command behaves this way.